TenureInformationApi/V1/Boundary/Requests/Validation/EditTenureDetailsRequestValidation.cs

```diff
@@ -10,9 +10,12 @@ public class EditTenureDetailsRequestValidation : AbstractValidator<EditTenureDe
     {
         public EditTenureDetailsRequestValidation()
         {
-            RuleFor(x => x.EndOfTenureDate)
+            When(tenure => tenure.EndOfTenureDate != null, () =>
+            {
+                RuleFor(x => x.EndOfTenureDate)
                .GreaterThan(x => x.StartOfTenureDate)
                .WithErrorCode(ErrorCodes.TenureEndDate);
+            });
```

Comment:
Is this ok when the start date is null? If it is maybe just add a test to prove the point.

Reply:
You're right. If there was no tenureStartDate, it would throw an error. I have changed the rule to require both startDate and endDate.